src/components/LayerActionIcon/LayerActionIcon.tsx

```
@@ -25,22 +25,19 @@ export function LayerActionIcon({
-      <TooltipContent>{hint}</TooltipContent>
-    </Tooltip>
+    <SimpleTooltip content={hint} placement="top" open={isOpen} onOpenChange={setIsOpen}>
+      <div
+        onFocus={() => setIsOpen(true)}
+        onBlur={() => setIsOpen(false)}
+        onMouseEnter={() => setIsOpen(true)}
+        onMouseLeave={() => setIsOpen(false)}
+        onKeyUp={onKeyUp}
+        className={clsx(style.actionButton, className)}
+        onClick={onClick}
+        tabIndex={0}
```

Biome (1.9.4)

[error] The HTML element div is non-interactive. Do not use tabIndex.

Adding non-interactive elements to the keyboard navigation flow can confuse users.
Unsafe fix: Remove the tabIndex attribute.


(lint/a11y/noNoninteractiveTabindex)